central/reports/scheduler/schedule.go

```diff
@@ -310,25 +316,21 @@ func (s *scheduler) sendReportResults(req *ReportRequest) error {
 	return nil
 }
 
-func formatMessage(rc *storage.ReportConfiguration) (string, error) {
+func formatMessage(rc *storage.ReportConfiguration, emailTemplate string, timestr string) (string, error) {
```

Comment:
... and apply formatting inside of this function
```go
func formatMessage(rc *storage.ReportConfiguration, emailTemplate string, date Time) (string, error) {
	data := &reportEmailFormat{
		// ...
		DateStr: date.Format("January 02, 2006"),
	}
	// ...
```

This way `formatMessage` will control date formatting and you will still be able to unit-test it with a fixed date.

Reply:
Done